Websocket Error 1006: http4s Backend Timout After 60 Secs

### Runtime
Version: Docspell 0.41.0 (via docspell/restserver:latest)
Browser: Firefox 108 / Chrome 120
Reverse proxy: Lighttpd 1.4.52 with `mod_proxy` and long-running websocket timeouts using the following config

```text
$HTTP["host"] == "subdomain.mydomain.com" {
  $HTTP["url"] == "/api/v1/sec/ws" {
        proxy.server  = ("" => (("host" => "docspell-restserver", "port" => 7880)))
        proxy.header = ( "upgrade" => "enable", "connect" => "enable")
        server.max-read-idle = 86400
        server.max-write-idle = 86400
        server.max-keep-alive-idle = 86400
      }
      else {
          $HTTP["url"] =~ "^/([^.](.*))$|^/$" {
          proxy.server  = ("" => (("host" => "docspell-restserver", "port" => 7880)))
      }
    }
        accesslog.filename = "/var/log/lighttpd/docspell.log"
}
```

### Expected behavior
Websocket connection working and pushing messages between frontend and backend.

### Actual behavior
Websocket connection timed out after 60 seconds after connection had been upgraded. The previous release v0.40.0 didn't run into this issue with same reverse proxy setup.

### Logs
```text
2024.02.15 13:09:30:0000 [io-comp...] [ERROR] org.http4s.ember.server.EmberServerBuilderCompanionPlatform - WebSocket connection terminated with exception
java.util.concurrent.TimeoutException: 60 seconds
        at timeout @ org.http4s.ember.core.Util$.timeoutMaybe(Util.scala:106)
        at main$ @ docspell.restserver.Main$.main(Main.scala:14)
        at timeout @ org.http4s.ember.core.Util$.timeoutMaybe(Util.scala:106)
        at timeout @ org.http4s.ember.core.Util$.timeoutMaybe(Util.scala:106)
```

![image](https://github.com/eikek/docspell/assets/13854802/1a283bea-f033-4cae-b14d-31126638936a)

--- a/modules/webapp/src/main/webjar/docspell.js
+++ b/modules/webapp/src/main/webjar/docspell.js
@@ -152,6 +152,8 @@ function initWS() {
             var dataJSON = JSON.parse(event.data);
             if (dataJSON.tag !== "keep-alive") {
                 elmApp.ports.receiveWsMessage.send(dataJSON);
+            } else {
+                dsWebSocket.send(event.data);
             }
         }
     });

